certbot-apache/certbot_apache/configurator.py

```diff
@@ -577,17 +581,12 @@ def _create_vhost(self, path):
         if filename is None:
             return None
 
-        if self.conf("handle-sites"):
-            is_enabled = self.is_site_enabled(filename)
```

Comment:
This is now addressed along with the fix for creating vhosts in a non-parsed directory (eg. original vhost in Apache root configuration). 

However we'll prefer vhost-root setting for newly created SSL vhosts from now on if the path exists, enabling (or adding Include in root config) if needed. This should make it usable in even more exotic configurations.